plugins/Kaleidoscope-Hardware-Keyboardio-Model100/src/kaleidoscope/device/keyboardio/Model100.h

```diff
@@ -17,7 +17,13 @@
 
 #pragma once
 
-#ifdef ARDUINO_GD32_keyboardio_model_100
+#ifdef ARDUINO_keyboardio_model_100
+
+#define _INIT_HID_GETSHORTNAME // warning "TODO: remove this directive once we have HID"
```

Comment:
Since we appear to have a HID, KeyboardioHID, even, should this be removed?

Reply:
Fixed.